src/Filme.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Filme.hpp"
+
+Filme::Filme (int _unidades, int _identificador, std::string _titulo) : 
+    unidades(_unidades), identificador(_identificador), titulo(_titulo) {}
+
+
+DVD::DVD (int _unidades, int _identificador, std::string _titulo, int _categoria) : 
+    Filme (_unidades, _identificador, _titulo) {
+        this->categoria = _categoria;
```

Comment:
Ao ínves de definir a categoria usando o this, sugiro usar uma fila de construtores antes do bloco de chaves, assim se evita que o construtor defina a varival com conteudo padrão